modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/management_bar/components/MainSearch.js

```diff
@@ -36,55 +36,44 @@ function MainSearch() {
 	}
 
 	return (
-		<div className="d-inline">
-			<div className="input-group">
-				<div className="input-group-item">
-					<div className="main-input-wrapper">
-						<input
-							className="form-control input-group-inset input-group-inset-after main-input-search"
-							onChange={(event) =>
-								updateInputValue(event.target.value)
-							}
-							onKeyDown={handleKeyDown}
-							placeholder={Liferay.Language.get('search')}
-							type="text"
-							value={inputValue}
-						/>
+		<ClayInput.Group>
```

Comment:
And maybe we could simply use `ClayManagementToolbar.Search` instead of custom markup.

Reply:
> And maybe we could simply use ClayManagementToolbar.Search instead of custom markup.

I'm working on just that to fix https://issues.liferay.com/browse/LPS-119865 :)

Reply:
Hey @brunobasto and @jbalsas, see 9569658.

I adapted Bruno's WIP from https://github.com/diegonvs/liferay-portal/pull/38 and updated.